web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -413,17 +404,13 @@ private static void setOncogenic(Gene gene, Alteration alteration, Oncogenicity
                 evidence.setGene(gene);
                 evidence.setAlterations(Collections.singleton(alteration));
                 evidence.setEvidenceType(EvidenceType.ONCOGENIC);
-                if (oncogenic != null) {
```

Comment:
because the oncogenic check is already done in Line 398
if (alteration != null && gene != null && oncogenic != null) {